configure.py

```diff
@@ -709,8 +709,13 @@ def get_llvm_version(cc):
     cc, r"(^(?:FreeBSD )?clang version|based on LLVM) ([3-9]\.[0-9]+)")
 
 def get_xcode_version(cc):
-  return get_version_helper(
+  version = get_version_helper(
     cc, r"(^Apple LLVM version) ([0-9]+\.[0-9]+)")
```

Comment:
You could do this more succinctly with a non-capturing group: `(?:clang|LLVM)`

Reply:
TIL you can nest capture groups. Thanks, I've updated the regex.